### ISSUE TITLE
Command stop charging only stops for 10 minutes

### ISSUE THREAD

**Issue by bosmanrja** [Hyundai-Kia-Connect/kia_uvo#893]
I use the integration with my Kia CEED. I want to control charging moments related to actual energy price. For this I need the stop charging command and start charging command. Both commands do work but when I stop charging I got the reply that this command is only valid for 10 minutes. After 10 minutes the charging proceed automatically. Someone has a solution for this?
Possibly I send the command every 10 minutes but this is probably not the best solution.

Thanks in advance.

Jan-Roel Bosman

**Comment by bosmanrja** [comment]
Yes it does. I checked and approx 1 minute after stop command charging was finished it starts charging again.

**Comment by cdnninja** [comment]
If working like the native app not much can be done.

**Comment by cdnninja** [comment]
If native app does this maybe open a ticket with Hyundai/Kia?

**Comment by bosmanrja** [comment]
I found out the way remote charging does work. First you need to make a schedule in your app.

I scheduled a charging moment between 03:00 and 06:00. Then I connect the car so it is actually waiting for this moment.
In this state I can control the charging b.e. based on the actual tariff.

As wel start as stop are functional now.
